screwdriver.yaml

```diff
@@ -1,19 +1,21 @@
 shared:
-    image: centos
+    image: golang:1.13
+    environment:
```

Comment:
golang version 1.13 do changed GO111MODULE default(auto mode) work.
I think so sould if not using go module function, GO111MODULE expressly set value off.

reference: https://github.com/golang/go/issues/31857

Reply:
Thanks, I set `GO111MODULE` to off explicitly.